### ISSUE TITLE
Use BlockDisplays for glowing blocks (when possible)

### ISSUE THREAD

**Issue by TheDeafCreeper** [SkytAsul/GlowingEntities#19]
Not all items work in BlockDisplays unfortunately, namely banners (maybe you could do some magic with ItemDisplays?), but it would allow much more flexibility for the shape of the glowing entity if a BlockDisplay was used.

![image](https://github.com/user-attachments/assets/8b6148cc-9743-406f-8f31-5ad2cd089511)

**Comment by SkytAsul** [comment]
That's right! But then what is the point of using GlowingBlock? The spigot
API must be more straightforward, no?

Le ven. 26 juil. 2024, 01:07, TheDeafCreeper ***@***.***> a
écrit :

**Comment by SkytAsul** [comment]
Yep I understand. If somebody wants to contribute this, I'll happily review but for now I don't have time.
